configs/solo/decoupled_solo_r50_fpn_8gpu_1x.py

```diff
@@ -0,0 +1,126 @@
+# model settings
```

Comment:
For config filenames, we usually use `8x2` rather than `8gpu` to also indicate the batch size as indicated [here](https://github.com/open-mmlab/mmdetection/blob/master/docs/config.md). If the config uses the default setting, i.e., 8 GPUS with 2 batch size per GPU, we can omit `8x2` or `8gpu` for simplicity.